packages/server/api.js

```diff
@@ -1384,7 +1360,7 @@ module.exports = async function attachAPI(app, {wss, db, dbDir}) {
 
       if (sessionUser._id === userID) {
         response.status(400).end(JSON.stringify({
-          error: 'you cannot deauthorize yourself'
+          error: errors.AUTHORIZATION_ERROR
```

Comment:
Ehh... this is a mix between `NOT_YOURS` and `AUTHORIZATION_ERROR`, isn't it? Could you use a custom `message` here to clarify?

Reply:
Sure, I'll add a custom message. I didn't put too much thought into AUTHORIZATION_ERRORs because they're going to be obsolete pretty near in the future.